svsim/src/main/scala/Workspace.scala

```diff
@@ -36,7 +36,24 @@ object Workspace {
       FileVisitResult.CONTINUE
     }
   }
+
+  /**
+  * Detect the project root (workspace) directory. It uses the "chisel.project.root" Java system property if it is set.
+  *   Otherwise, checks the environment variables CHISEL_PROJECT_ROOT (can be set by user) and MILL_WORKSPACE_ROOT (set by mill).
+  * If none of those work, returns the current working directory.
+  *
+  * @return the absolute path to the project root or the current working directory
+  */
```

Comment:
Nit: this comment syntax is non-standard with the rest of the project. I expected:

``` scala
/** Detect...
  * Otherwise
  */
```

Reply:
Thanks! Fixed.